app/Jobs/SpawnQueryserviceUpdaterJob.php

```diff
@@ -0,0 +1,115 @@
+        $this->qsKubernetesNamespace = env('QS_JOB_NAMESPACE', 'qs-jobs');
+    }
+
+    public function uniqueId(): string
+    {
+        return $this->wikiDomain;
+    }
+
+    public function handle (Client $kubernetesClient): void
+    {
+        $kubernetesClient->setNamespace('default');
+        $qsUpdaterPod = $kubernetesClient->pods()->setFieldSelector([
+            'status.phase' => 'Running'
+        ])->setLabelSelector([
+            'app.kubernetes.io/name' => 'queryservice-updater',
+        ])->first();
+
+        if ($qsUpdaterPod === null) {
+            $this->fail(
+                new \RuntimeException(
+                    'Unable to find a running queryservice-updater pod in the cluster, '.
+                    'cannot continue.'
+                )
+            );
+            return;
+        }
+        $qsUpdaterPod = $qsUpdaterPod->toArray();
+
+        $kubernetesClient->setNamespace($this->qsKubernetesNamespace);
+        $jobSpec = new KubernetesJob([
+            'metadata' => [
+                'name' => 'run-qs-updater-'.hash('sha1', $this->wikiDomain),
+                'namespace' => $this->qsKubernetesNamespace,
+                'labels' => [
+                    'app.kubernetes.io/instance' => $this->wikiDomain,
+                    'app.kubernetes.io/name' => 'run-qs-updater'
+                ]
+            ],
+            'spec' => [
+                'ttlSecondsAfterFinished' => 14 * 24 * 60 * 60, // 2 weeks
+                'template' => [
+                    'metadata' => [
+                        'name' => 'run-qs-updater'
+                    ],
+                    'spec' => [
+                        'containers' => [
+                            0 => [
+                                'name' => 'run-qs-updater',
+                                'image' => $qsUpdaterPod['spec']['containers'][0]['image'],
+                                'env' => $qsUpdaterPod['spec']['containers'][0]['env'],
+                                'command' => [
+                                    0 => 'bash',
+                                    1 => '-c',
+                                    2 => <<<CMD
+                                    /wdqsup/runUpdateWbStack.sh -- \
+                                        --wikibaseHost {$this->wikiDomain} \
+                                        --ids {$this->entities} \
+                                        --entityNamespaces 120,122,146 \
+                                        --sparqlUrl {$this->sparqlUrl} \
+                                        --wikibaseScheme http \
+                                        --conceptUri https://{$this->wikiDomain}
+                                    CMD
+                                ],
+                            ]
+                        ],
+                        'restartPolicy' => 'Never'
+                    ]
+                ]
+            ]
+        ]);
+
+        $job = $kubernetesClient->jobs()->apply($jobSpec);
```

Comment:
Should this be `create` instead (making it fail when the job already exists). Not sure.

Reply:
probably depends on if we think jobs should be unique per wiki or not (e.g. in the event one fails and we try again)

Reply:
Maybe we can decide this when implementing the command. Right now I would think it doesn't matter, but then I don't know which retry mechanisms we come up with lateron.